lib/smart_proxy_remote_execution_ssh/mqtt/dispatcher.rb

```diff
@@ -2,15 +2,39 @@
 require 'mqtt'
 
 class Proxy::RemoteExecution::Ssh::MQTT
+  class DispatcherSupervisor < Concurrent::Actor::RestartingContext
+    def initialize()
```

Comment:
That was a long time I've seen a Ruby method definition with empty param list using brackets :)